bugbug/commit_features.py

```diff
@@ -478,26 +433,18 @@ def __call__(self, commit, **kwargs):
                 "halstead_n1_total"
             ],
             "Total function number of operators": merged_metrics["halstead_N1_total"],
-            "Total function length": merged_metrics["halstead_length_total"],
-            "Total calculated estimated function length": merged_metrics[
+            "Total file length": merged_metrics["halstead_length_total"],
+            "Total file estimated program length": merged_metrics[
```

Comment:
I think the names are already correct (the same for others). See the name of the function:
https://github.com/mozilla/bugbug/blob/c96c2e2f03a9a03c73f67c3aa1583779b06c42d5/bugbug/commit_features.py#L335